airgun/browser.py

```diff
@@ -135,17 +129,14 @@ def post_init(self):
         :return: None
         """
         # Workaround 'Certificate Error' screen on Microsoft Edge
-        if (
-                self.browser == 'edge' and
-                ('Certificate Error' in self._webdriver.title or
-                    'Login' not in self._webdriver.title)):
+        if self.browser == 'edge' and (
+            'Certificate Error' in self._webdriver.title or 'Login' not in self._webdriver.title
+        ):
             self._webdriver.get(
-                "javascript:document.getElementById('invalidcert_continue')"
-                ".click()"
+                "javascript:document.getElementById('invalidcert_continue')" ".click()"
```

Comment:
```suggestion
                "javascript:document.getElementById('invalidcert_continue').click()"
```

Reply:
I looked for an found other instances of this, they should be fixed in the latest commit.